psi4/src/psi4/ccresponse/scatter.cc

```diff
@@ -301,7 +301,7 @@ void scatter(std::shared_ptr<Molecule> molecule, Options &options, double step,
     // Mass-weighting the co-ordinates //
     //double massi[natom];
     outfile->Printf("\tAtomic Masses for Raman Computation:\n");
-    for(i=0; i < natom; i++) outfile->Printf("\t%d %12.8f\n", i, molecule->mass(i));
+    for(i=0; i < natom; i++) outfile->Printf("\t%d %12.8f\n", i, molecule->mass(i,true));
```

Comment:
Mass calls in `scatter.cc` are for molecular weights or mass weighting, which probably means they do not intend to include the masses of ghost atoms.